### Manejo de excepciones (o manejo de errores) ###

number_one = 5
number_two = 1
number_two = "1"

print(number_one + number_two)